### PR TITLE
[CloudPC]Onboard new canary error type: UdpConnectivityCheckStunUrlNotAllowListed,UdpConnectivityCheckUnknownError

### PR DESCRIPTION
**Pull request by Emma0727**
[CloudPC]Onboard new canary error type: UdpConnectivityCheckStunUrlNotAllowListed,UdpConnectivityCheckUnknownError
Work item ADO: Feature 15431

### CODE REVIEW

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@Emma0727 
1.Please also update the changelog/Microsoft.CloudManagedDesktop.json. 
2. Please create a roadmap workitem and attach the workitem number in comment. Here is an [example](https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/14460/). If you don't have access, please do these firstly:
- Request access via [Improved ADO Access - 1ES Wikirequest to join](https://www.1eswiki.com/wiki/Improved_ADO_Access) .
- Join the Microsoft Graph Roadmap Users ([graphroadmap@microsoft.com](mailto:graphroadmap@microsoft.com)) security group on [idweb](https://idwebelements/GroupManagement.aspx?Group=graphroadmap&Operation=join) .

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]
Please add change-log @Emma0727

**Review by AshleyYangSZ — APPROVED** [review]

**Review by RuiHou105 — APPROVED** [review]

File: api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md [hunk 1]
```diff
@@ -92,6 +92,8 @@ The result of a Cloud PC Azure network connection health check.
 |permissionCheckNoResourceGroupNetworkContributorRole|Cloud PC service principal doesn't have sufficient permissions on the Azure resource group. Please make sure that the  application has Network contributor permissions on the resource group.|
 |permissionCheckTransientServiceError|The first-party app permission check failed due to a transient error. Please try it again. If the issue persists, please contact customer support.|
 |permissionCheckUnknownError|The Cloud PC service principal doesn't have sufficient permissions. Please make sure that the Cloud PC service principal is granted sufficient Azure permissions.|
+|udpConnectivityCheckStunUrlNotAllowListed|Your current network configuration does not allow the use of UDP direct connect (STUN). This will not prevent the use of Cloud PCs but may prevent optimal performance. Consider your own network configuration policies before making changes.|
```
Comment (dodaromike) [inline api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md:95]: The acronym STUN is not defined.  Please add definition.

Comment (isvargasmsft) [reply api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md:95]: Following this comment, please also add the definition of UDP.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @Emma0727, 

Thank you very much for your work in this PR. Very complete. 

We just wanted to have a definition of the STUN acronym that is not clear to us. Could you please add the definition of the acronym? 

Please feel free to ping me on the side if you have questions. 

Kind regards, 

Isaac and Mike.

File: api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md (hunk 1 above)
Comment (Emma0727) [reply api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md:95]: sorry, I missed this comment, will add definition.

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**